app/controllers/development_metrics_controller.rb

```diff
@@ -1,5 +1,6 @@
 class DevelopmentMetricsController < ApplicationController
   layout 'sidebar_metrics'
+  include Concerns::LoadSettings
```

Comment:
is this concerns going to be reused? Or why did you use a concern and not just put the logic in this controller?

Reply:
Since there may appear more settings in the future, I think it's a good idea to place this on a concern. I don't quite like this on the controller.

Reply:
Actually yes, it's used in several controllers.